bin/setup.sh

```diff
@@ -81,6 +81,11 @@ function do_gencode() {
 }
 
 ##############################
+function default_settings() {
```

Comment:
maybe using a name that reflects what it does would be better, like `set_default_settings`